Filled out (0x33) Chunk Information

OfficialPixelBrush opened the pull request:
- Added Clientbound info
- Added zlib and deflate spec references
- Added example data

OfficialPixelBrush committed:
Filled chunk.md

- Added Clientbound info
- Added zlib and deflate spec references
- Added example data

OfficialPixelBrush committed:
test

Trying to figure out Jekyll

getting there?

Maybe? Theme switch either way

Idk????

grrrr

Spacing?

Trying full tables?

Are collections our savior?

maybe???

idfk

whoops?

uhhh idk i fucked it?

aaaaaa

?????

More, maybe?

aaaaa

????

idfk

MAYBE????

progress, I think

idfk man

Update _config.yml

man

Update _config.yml

????????

whoops

ah

AAAAAAAAAAAA

Getting somewhere

Push

OfficialPixelBrush committed:
Personal preference + Signs

OfficialPixelBrush committed:
more stuff

- Dark mode by default
- Rendering info (WIP)
- World loading info (WIP)
- Player position

OfficialPixelBrush committed:
Improved login

OfficialPixelBrush committed:
More stuff

OfficialPixelBrush committed:
Whoops + Mining

OfficialPixelBrush committed:
Notices

OfficialPixelBrush committed:
Update todo.md

OfficialPixelBrush committed:
better colors

OfficialPixelBrush committed:
Very nice

OfficialPixelBrush committed:
btt

OfficialPixelBrush committed:
Lots of notes + NBT

OfficialPixelBrush committed:
more stuff

OfficialPixelBrush committed:
wip

OfficialPixelBrush committed:
blocks

OfficialPixelBrush committed:
Mobs

typo!

OfficialPixelBrush committed:
Reorganizing + In-game block names

Update index.md

Yeah, cool

mudkipdev commented:
Hi, I like your idea of turning it into a site. Is there anywhere we can chat or on Discord?

mudkipdev commented:
I added a site here for now (including explanations for entity metadata): https://beta-wiki.pages.dev
I also have this wiki page which was given to me, I am not sure how it compares to your documentation but it has all block IDs and metadata explanations

[Data_values.html.zip](https://github.com/user-attachments/files/21583375/Data_values.html.zip)


Some more miscellaneous resources: https://github.com/PhoenixVX/golden-age-wiki

OfficialPixelBrush commented:
> Hi, I like your idea of turning it into a site. Is there anywhere we can chat or on Discord?

I‘d have Discord. I'm just `pixelbrush` on there.

OfficialPixelBrush commented:
Also, I admit I forgot that this is how GitHub handles PRs. Didn’t think it‘d show the commits I‘ve made since, haha. Should’ve probably done the website stuff on a separate branch or something 😅

OfficialPixelBrush committed:
Basic generation history

OfficialPixelBrush committed:
Zooming and more images

OfficialPixelBrush committed:
Noise + Random

OfficialPixelBrush committed:
Fixed Link on NBT Page

OfficialPixelBrush committed:
Noise and Farlands

OfficialPixelBrush committed:
Fixed path

OfficialPixelBrush committed:
Simplex

OfficialPixelBrush committed:
Reorder and clarification

OfficialPixelBrush committed:
Lots on Biomes + Color Swatch support

OfficialPixelBrush committed:
Top + Filler Blocks for Biomes + Better noise names

OfficialPixelBrush committed:
Noise names

OfficialPixelBrush committed:
Reorg and McRegion stub + Favicon!

OfficialPixelBrush committed:
Removal of Voxatile. Wishing you luck with Uni!

OfficialPixelBrush committed:
Reorg, more images, better var names, terrain stuff

OfficialPixelBrush committed:
Improved code clarity

OfficialPixelBrush committed:
Range -1,-1 to 1,1 + Deco noises

- More readable (albeit less accurate) terrain images
- Sand, Gravel and Stone Noise maps
- Improved Temperature, Humidity and Weirdness maps

OfficialPixelBrush committed:
Math + Cave slot

OfficialPixelBrush committed:
Final Biome images + Centered images

OfficialPixelBrush committed:
Better biome names

OfficialPixelBrush committed:
Stub

OfficialPixelBrush commented:
These pages have diverged a bit too far to get merged cleanly, so to stop you from getting constant notifications I'll finally just close the PR.
